client/src/context/user/userReducer.js

```diff
@@ -0,0 +1,25 @@
+import { GET_USER } from '../types';
+
+const contactReducer = (state, action) => {
+    switch (action.type) {
+        case GET_USER:
+            return {
+                ...state, //TODO: might not need, might not work as intended

```

Comment:
Yeah this feels a bit strange.

First off, I think you can just return a copy of the current state as this code currently stands. What you have now seems to be a little redundant. i.e. This could become jus this instead.
```
return {
    ...state,
}
```

Secondly, is this a partial implementation? We'd likely want to receiving some sort of user data on this action to update the state with 🤔